src/core/Modal/BaseModal/BaseModal.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { ThemedBackground } from '../../../utils/storybook';
+import BaseModal from '../BaseModal/BaseModal';
+
+storiesOf('Modal/Base Modal', module)
```

Comment:
Since we only have one Modal and there are no variants or types for this, I think just keeping `Modal` would be a better choice for this component.